pristine-zen-garden.html

```diff
@@ -0,0 +1,176 @@
+      </div>
+
+      <footer>
+        <a href="http://validator.w3.org/check/referer" title="Check the validity of this site&#8217;s HTML" class="zen-validate-html">HTML</a>
+        <a href="http://jigsaw.w3.org/css-validator/check/referer" title="Check the validity of this site&#8217;s CSS" class="zen-validate-css">CSS</a>
+        <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/" title="View the Creative Commons license of this site: Attribution-NonCommercial-ShareAlike." class="zen-license">CC</a>
+        <a href="http://mezzoblue.com/zengarden/faq/#aaa" title="Read about the accessibility of this site" class="zen-accessibility">A11y</a>
+        <a href="https://github.com/mezzoblue/csszengarden.com" title="Fork this site on Github" class="zen-github">GH</a>
+      </footer>
+
+    </div>
+
+    <aside class="sidebar" role="complementary">
+
+      <div class="wrapper">
+
+        <div class="design-selection" id="design-selection">
+
+          <h3 class="select">Select a Design:</h3>
+
+          <nav role="navigation">
+
+            <ul>
+              <li>
+                <a href="/205/" class="design-name">spring360</a> by <a href="http://www.medialab360.com" class="designer-name">Rene Hornig</a>
+              </li>
+              <li>
+                <a href="/204/" class="design-name">Withering Beauty</a> by <a href="http://www.wduffy.co.uk/" class="designer-name">William Duffy</a>
+              </li>
+              <li>
+                <a href="/203/" class="design-name">Tiny Blue</a> by <a href="http://www.timovirtanen.com/" class="designer-name">Timo Virtanen</a>
+              </li>
+              <li>
+                <a href="/202/" class="design-name">Retro Theater</a> by <a href="#" class="designer-name">Eric Rogé</a>
+              </li>
+              <li>
+                <a href="/201/" class="design-name">Lily Pond</a> by <a href="#" class="designer-name">Rose Thorogood</a>
+              </li>
+              <li>
+                <a href="/200/" class="design-name">Icicle Outback</a> by <a href="http://www.timovirtanen.com/" class="designer-name">Timo Virtanen</a>
+              </li>
+              <li>
+                <a href="/199/" class="design-name">CSS ZEN ARMY</a> by <a href="http://www.niceguy.com" class="designer-name">Carl Desmond</a>
+              </li>
+              <li>
+                <a href="/198/" class="design-name">The Original</a> by <a href="#" class="designer-name">Joachim Shotter</a>
+              </li>
+            </ul>
+
+          </nav>
+
+        </div>
+
+        <div class="design-archives" id="design-archives">
+
+          <h3 class="archives">Archives:</h3>
+
+          <nav role="navigation">
+
+            <ul>
+              <li class="next">
+                <a href="/214/page3">
+                Next Designs <span class="indicator">&rsaquo;</span>
```

Comment:
Thanks; that was more an unfortunate line break, as opposed to indention. Will edit to look better